website/docs/docs/deploy/hybrid-setup.md

```diff
@@ -0,0 +1,159 @@
+    - [Configure dbt Core project and upload artifacts](#configure-dbt-core-project-and-upload-artifacts)
+    - [Review artifacts in dbt Cloud](#review-artifacts-in-dbt-cloud)
+
+Make sure to enable the hybrid projects toggle in dbt Cloud’s **Account settings** page.
+
+### Make dbt Core models public
+
+Before connecting your dbt Core project to a dbt Cloud project, make sure models that you want to share have `access: public` in their model configuration. This setting makes those models visible to other dbt Cloud projects for better collaboration, such as [cross-project referencing](/docs/collaborate/govern/project-dependencies#how-to-write-cross-project-ref).
+
+1. The easiest way to set this would be in your `dbt_project.yml` file, however you can also set this in the following places:
+   - `dbt_project.yml` (project-level)
+   - `properties.yml` (for individual models)
+   - A model's `.sql` file using a `config` block
+
+   Here's an example using a `dbt_project.yml` file where the marts directory is set as public so they can be consumed by downstream tools:
+  
+   <File name='dbt_project.yml'>
+
+   ```yaml
+   models:
+   define_public_models: # This is my project name, remember it must be specified
+   marts:
+      +access: public
+   ```
+   </File>
+
+2. After defining `access: public`, rerun a dbt execution in the command line interface (CLI) (like `dbt run`) to apply the change.
+
+3. For more details on how to set this up, see [access modifier](/docs/collaborate/govern/model-access#access-modifiers) and [`access` config](/reference/resource-configs/access). 
+
+### Create hybrid project
+
+Create a hybrid project in dbt Cloud to allow you to upload your dbt Core artifacts to dbt Cloud. 
+
+A [dbt Cloud account admin](/docs/cloud/manage-access/enterprise-permissions#permission-sets) should perform the following steps and share the artifacts information with a dbt Core user:
+
+1. To create a new project in dbt Cloud, navigate to **Account home**.
+2. Click on **+New project**. 
+3. Fill out the **Project name**. Name the project something that allows you to recognize it's a dbt Core project. 
+   - You don't need to set up a [data warehouse](/docs/supported-data-platforms) or [Git connection](/docs/cloud/git/git-configuration-in-dbt-cloud), however to upgrade the hybrid project to a full dbt Cloud project, you'd need to set up data warehouse and Git.
+4. Select the **Advanced settings** toggle and then select the **Hybrid development** checkbox. Click **Continue**.
+   - The hybrid project will have a visible **Hybrid** indicator in the project list to help you identify it.
+<Lightbox src="/img/docs/deploy/hp-new-project.jpg" title="Hybrid project new project" />
+
+5. After creating a project, create a corresponding [production environment](/docs/deploy/deploy-environments#create-a-deployment-environment) and click **Save**.
```

Comment:
nit, one thing to maybe callout here is that the connection field can be left blank, the only thing we need is the environment to be a production environment.